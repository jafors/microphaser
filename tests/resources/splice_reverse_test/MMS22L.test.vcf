##fileformat=VCFv4.1
##FILTER=<ID=PASS,Description="All filters passed">
##fileDate=20181121
##source=freeBayes v0.9.21-7-g7dd41db
##reference=/vol/tiny/ref/autoref/Homo_sapiens/NCBI/GRCh38/BWAIndex/genome.fa
##phasing=none
##commandline="freebayes -f /vol/tiny/ref/autoref/Homo_sapiens/NCBI/GRCh38/BWAIndex/genome.fa out/bam/SRR3114444.bam out/bam/SRR3114519.bam out/bam/SRR3114558.bam out/bam/SRR3114559.bam out/bam/SRR3114565.bam out/bam/SRR3114566.bam out/bam/SRR3114573.bam out/bam/SRR3114577.bam out/bam/SRR3114578.bam out/bam/SRR3114580.bam --min-alternate-fraction 0.10 --region chr1:0-10000000"
##filter="QUAL > 10"
##INFO=<ID=NS,Number=1,Type=Integer,Description="Number of samples with data">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Total read depth at the locus">
##INFO=<ID=DPB,Number=1,Type=Float,Description="Total read depth per bp at the locus; bases in reads overlapping / bases in haplotype">
##INFO=<ID=AC,Number=A,Type=Integer,Description="Total number of alternate alleles in called genotypes">
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
##INFO=<ID=AF,Number=A,Type=Float,Description="Estimated allele frequency in the range (0,1]">
##INFO=<ID=RO,Number=1,Type=Integer,Description="Reference allele observation count, with partial observations recorded fractionally">
##INFO=<ID=AO,Number=A,Type=Integer,Description="Alternate allele observations, with partial observations recorded fractionally">
##INFO=<ID=PRO,Number=1,Type=Float,Description="Reference allele observation count, with partial observations recorded fractionally">
##INFO=<ID=PAO,Number=A,Type=Float,Description="Alternate allele observations, with partial observations recorded fractionally">
##INFO=<ID=QR,Number=1,Type=Integer,Description="Reference allele quality sum in phred">
##INFO=<ID=QA,Number=A,Type=Integer,Description="Alternate allele quality sum in phred">
##INFO=<ID=PQR,Number=1,Type=Float,Description="Reference allele quality sum in phred for partial observations">
##INFO=<ID=PQA,Number=A,Type=Float,Description="Alternate allele quality sum in phred for partial observations">
##INFO=<ID=SRF,Number=1,Type=Integer,Description="Number of reference observations on the forward strand">
##INFO=<ID=SRR,Number=1,Type=Integer,Description="Number of reference observations on the reverse strand">
##INFO=<ID=SAF,Number=A,Type=Integer,Description="Number of alternate observations on the forward strand">
##INFO=<ID=SAR,Number=A,Type=Integer,Description="Number of alternate observations on the reverse strand">
##INFO=<ID=SRP,Number=1,Type=Float,Description="Strand balance probability for the reference allele: Phred-scaled upper-bounds estimate of the probability of observing the deviation between SRF and SRR given E(SRF/SRR) ~ 0.5, derived using Hoeffding's inequality">
##INFO=<ID=SAP,Number=A,Type=Float,Description="Strand balance probability for the alternate allele: Phred-scaled upper-bounds estimate of the probability of observing the deviation between SAF and SAR given E(SAF/SAR) ~ 0.5, derived using Hoeffding's inequality">
##INFO=<ID=AB,Number=A,Type=Float,Description="Allele balance at heterozygous sites: a number between 0 and 1 representing the ratio of reads showing the reference allele to all reads, considering only reads from individuals called as heterozygous">
##INFO=<ID=ABP,Number=A,Type=Float,Description="Allele balance probability at heterozygous sites: Phred-scaled upper-bounds estimate of the probability of observing the deviation between ABR and ABA given E(ABR/ABA) ~ 0.5, derived using Hoeffding's inequality">
##INFO=<ID=RUN,Number=A,Type=Integer,Description="Run length: the number of consecutive repeats of the alternate allele in the reference genome">
##INFO=<ID=RPP,Number=A,Type=Float,Description="Read Placement Probability: Phred-scaled upper-bounds estimate of the probability of observing the deviation between RPL and RPR given E(RPL/RPR) ~ 0.5, derived using Hoeffding's inequality">
##INFO=<ID=RPPR,Number=1,Type=Float,Description="Read Placement Probability for reference observations: Phred-scaled upper-bounds estimate of the probability of observing the deviation between RPL and RPR given E(RPL/RPR) ~ 0.5, derived using Hoeffding's inequality">
##INFO=<ID=RPL,Number=A,Type=Float,Description="Reads Placed Left: number of reads supporting the alternate balanced to the left (5') of the alternate allele">
##INFO=<ID=RPR,Number=A,Type=Float,Description="Reads Placed Right: number of reads supporting the alternate balanced to the right (3') of the alternate allele">
##INFO=<ID=EPP,Number=A,Type=Float,Description="End Placement Probability: Phred-scaled upper-bounds estimate of the probability of observing the deviation between EL and ER given E(EL/ER) ~ 0.5, derived using Hoeffding's inequality">
##INFO=<ID=EPPR,Number=1,Type=Float,Description="End Placement Probability for reference observations: Phred-scaled upper-bounds estimate of the probability of observing the deviation between EL and ER given E(EL/ER) ~ 0.5, derived using Hoeffding's inequality">
##INFO=<ID=DPRA,Number=A,Type=Float,Description="Alternate allele depth ratio.  Ratio between depth in samples with each called alternate allele and those without.">
##INFO=<ID=ODDS,Number=1,Type=Float,Description="The log odds ratio of the best genotype combination to the second-best.">
##INFO=<ID=GTI,Number=1,Type=Integer,Description="Number of genotyping iterations required to reach convergence or bailout.">
##INFO=<ID=TYPE,Number=A,Type=String,Description="The type of allele, either snp, mnp, ins, del, or complex.">
##INFO=<ID=CIGAR,Number=A,Type=String,Description="The extended CIGAR representation of each alternate allele, with the exception that '=' is replaced by 'M' to ease VCF parsing.  Note that INDEL alleles do not have the first matched base (which is provided by default, per the spec) referred to by the CIGAR.">
##INFO=<ID=NUMALT,Number=1,Type=Integer,Description="Number of unique non-reference alleles in called genotypes at this position.">
##INFO=<ID=MEANALT,Number=A,Type=Float,Description="Mean number of unique non-reference allele observations per sample with the corresponding alternate alleles.">
##INFO=<ID=LEN,Number=A,Type=Integer,Description="allele length">
##INFO=<ID=MQM,Number=A,Type=Float,Description="Mean mapping quality of observed alternate alleles">
##INFO=<ID=MQMR,Number=1,Type=Float,Description="Mean mapping quality of observed reference alleles">
##INFO=<ID=PAIRED,Number=A,Type=Float,Description="Proportion of observed alternate alleles which are supported by properly paired read fragments">
##INFO=<ID=PAIREDR,Number=1,Type=Float,Description="Proportion of observed reference alleles which are supported by properly paired read fragments">
##INFO=<ID=technology.Illumina,Number=A,Type=Float,Description="Fraction of observations supporting the alternate observed in reads from Illumina">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=GQ,Number=1,Type=Float,Description="Genotype Quality, the Phred-scaled marginal (or unconditional) probability of the called genotype">
##FORMAT=<ID=GL,Number=G,Type=Float,Description="Genotype Likelihood, log10-scaled likelihoods of the data given the called genotype for each possible genotype generated from the reference and alternate alleles given the sample ploidy">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
##FORMAT=<ID=RO,Number=1,Type=Integer,Description="Reference allele observation count">
##FORMAT=<ID=QR,Number=1,Type=Integer,Description="Sum of quality of the reference observations">
##FORMAT=<ID=AO,Number=A,Type=Integer,Description="Alternate allele observation count">
##FORMAT=<ID=QA,Number=A,Type=Integer,Description="Sum of quality of the alternate observations">
##SnpEffVersion="4.3t (build 2017-11-24 10:18), by Pablo Cingolani"
##SnpEffCmd="SnpEff  GRCh38.86 out/vcf/complete.vcf "
##INFO=<ID=ANN,Number=.,Type=String,Description="Functional annotations: 'Allele | Annotation | Annotation_Impact | Gene_Name | Gene_ID | Feature_Type | Feature_ID | Transcript_BioType | Rank | HGVS.c | HGVS.p | cDNA.pos / cDNA.length | CDS.pos / CDS.length | AA.pos / AA.length | Distance | ERRORS / WARNINGS / INFO'">
##INFO=<ID=LOF,Number=.,Type=String,Description="Predicted loss of function effects for this variant. Format: 'Gene_Name | Gene_ID | Number_of_transcripts_in_gene | Percent_of_transcripts_affected'">
##INFO=<ID=NMD,Number=.,Type=String,Description="Predicted nonsense mediated decay effects for this variant. Format: 'Gene_Name | Gene_ID | Number_of_transcripts_in_gene | Percent_of_transcripts_affected'">
##SnpSiftVersion="SnpSift 4.3r (build 2017-09-06 16:41), by Pablo Cingolani"
##SnpSiftCmd=""
##FILTER=<ID=SnpSift,Description="SnpSift 4.3r (build 2017-09-06 16:41), by Pablo Cingolani, Expression used: (!(ANN[*].HGVS_P has '' ) & QUAL > 10)">
##source_20181122.1=vcf-subset(0.953) -c SRR3114444,SRR3114565 out/vcf/complete_proteincoding.vcf
##contig=<ID=chr1>
##contig=<ID=chr2>
##contig=<ID=chr3>
##contig=<ID=chr4>
##contig=<ID=chr5>
##contig=<ID=chr6>
##contig=<ID=chr7>
##contig=<ID=chr8>
##contig=<ID=chr9>
##contig=<ID=chr10>
##contig=<ID=chr11>
##contig=<ID=chr12>
##contig=<ID=chr13>
##contig=<ID=chr14>
##contig=<ID=chr15>
##contig=<ID=chr16>
##contig=<ID=chr17>
##contig=<ID=chr18>
##contig=<ID=chr19>
##contig=<ID=chr20>
##contig=<ID=chr21>
##contig=<ID=chr22>
##contig=<ID=chrX>
##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description="Somatic variant">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SRR3114444	SRR3114565
chr6	97229242	.	G	A	38834.8	.	AB=0;ABP=0;AC=4;AF=1;AN=4;ANN=A|missense_variant|MODERATE|MMS22L|ENSG00000146263|transcript|ENST00000275053.8|protein_coding|14/25|c.1691C>T|p.Thr564Met|1957/8643|1691/3732|564/1243||,A|missense_variant|MODERATE|MMS22L|ENSG00000146263|transcript|ENST00000369251.6|protein_coding|12/23|c.1571C>T|p.Thr524Met|1647/3949|1571/3612|524/1203||,A|downstream_gene_variant|MODIFIER|MMS22L|ENSG00000146263|transcript|ENST00000509383.5|nonsense_mediated_decay||c.*41036C>T|||||1027|,A|downstream_gene_variant|MODIFIER|MMS22L|ENSG00000146263|transcript|ENST00000510018.5|protein_coding||c.*4619C>T|||||4619|WARNING_TRANSCRIPT_NO_START_CODON;AO=1165;CIGAR=1X;DP=1165;DPB=1165;DPRA=0;EPP=4.57786;EPPR=0;GTI=0;LEN=1;MEANALT=1;MQM=60;MQMR=0;NS=10;NUMALT=1;ODDS=119.985;PAIRED=0.996567;PAIREDR=0;PAO=0;PQA=0;PQR=0;PRO=0;QA=43984;QR=0;RO=0;RPL=678;RPP=71.0081;RPPR=0;RPR=487;RUN=1;SAF=297;SAP=610.726;SAR=868;SRF=0;SRP=0;SRR=0;TYPE=snp;technology.Illumina=1	GT:DP:RO:QR:AO:QA:GL	1/1:76:0:0:76:2830:-254.86,-22.8783,0	1/1:130:0:0:130:4904:-441.326,-39.1339,0
chr6	97231645	.	G	A	4665.3	.	AB=0;ABP=0;AC=2;AF=0.8;AN=4;ANN=A|missense_variant|MODERATE|MMS22L|ENSG00000146263|transcript|ENST00000275053.8|protein_coding|13/25|c.1310C>T|p.Ser437Phe|1576/8643|1310/3732|437/1243||,A|missense_variant|MODERATE|MMS22L|ENSG00000146263|transcript|ENST00000369251.6|protein_coding|11/23|c.1190C>T|p.Ser397Phe|1266/3949|1190/3612|397/1203||,A|3_prime_UTR_variant|MODIFIER|MMS22L|ENSG00000146263|transcript|ENST00000509383.5|nonsense_mediated_decay|14/14|c.*875C>T|||||38633|,A|downstream_gene_variant|MODIFIER|MMS22L|ENSG00000146263|transcript|ENST00000510018.5|protein_coding||c.*2216C>T|||||2216|WARNING_TRANSCRIPT_NO_START_CODON;AO=154;CIGAR=1X;DP=218;DPB=218;DPRA=0.601562;EPP=8.65049;EPPR=3.14602;GTI=0;LEN=1;MEANALT=1;MQM=59.987;MQMR=60;NS=10;NUMALT=1;ODDS=2.26246;PAIRED=1;PAIREDR=0.984375;PAO=0;PQA=0;PQR=0;PRO=0;QA=5650;QR=2303;RO=64;RPL=105;RPP=47.2294;RPPR=19.4321;RPR=49;RUN=1;SAF=18;SAP=199.345;SAR=136;SRF=10;SRP=68.6973;SRR=54;TYPE=snp;technology.Illumina=1;SOMATIC	GT:DP:RO:QR:AO:QA:GL	1/1:21:0:0:21:686:-62.0126,-6.32163,0	0/0:27:27:980:0:0:0,-8.12781,-88.4998
