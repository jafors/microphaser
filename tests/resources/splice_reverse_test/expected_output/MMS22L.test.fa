>d30b9dac24c00d9R
AAGCCAAGAAATACTGAAGaAACTATT
>c225e293273e48fR
CCAAGAAATACTGAAGaAACTATTCAG
>7011a59454dda74R
AGAAATACTGAAGaAACTATTCAGGTT
>1efa508d42bd9bcR
AATACTGAAGaAACTATTCAGGTTCTT
>1fca23b4f4cddc8R
ACTGAAGaAACTATTCAGGTTCTTACT
>210cb330fa6bccfR
GAAGaAACTATTCAGGTTCTTACTATA
>302a1fb3856f228R
GaAACTATTCAGGTTCTTACTATAATA
>9209752f31e1497R
ACTATTCAGGTTCTTACTATAATATTC
>4f610280fdd88a2R
AGGAAGCCAAGAAATACTGAAGaAACT
>96db41696eab759R
AAAAGGAAGCCAAGAAATACTGAAGaA
>2b53a11462068baR
CaTTACAAAAGCAGGCTTGAGGAAATT
>bdaef3c47fab5c3R
AGACaTTACAAAAGCAGGCTTGAGGAA
>b342cc839483116R
CTGAGACaTTACAAAAGCAGGCTTGAG
>dde4785f31c14aeR
TCTCTGAGACaTTACAAAAGCAGGCTT
>9a11a0c3d529fd0R
GGCTCTCTGAGACaTTACAAAAGCAGG
>1ef3e87b4a95bdfR
GAGGGCTCTCTGAGACaTTACAAAAGC
>7db10c9ecbd341cR
AATGAGGGCTCTCTGAGACaTTACAAA
>165b1f328bc037cR
CCAAATGAGGGCTCTCTGAGACaTTAC
>c805e1b269549fbR
CTTCCAAATGAGGGCTCTCTGAGACaT
>2df200b2e3eb7d3R
AAGCCAAGAAATACTGAAGaAACTATT
>6f698c636022ea8R
CCAAGAAATACTGAAGaAACTATTCTG
>5de706d492a4e33R
AGAAATACTGAAGaAACTATTCTGAAC
>8420c14e9383865R
AATACTGAAGaAACTATTCTGAACACT
>997cb4e940b6d5fR
ACTGAAGaAACTATTCTGAACACTGAT
>24362f4a2ca7fc6R
GAAGaAACTATTCTGAACACTGATGGA
>434939c4c42b55cR
GaAACTATTCTGAACACTGATGGACTT
>e75bf7f0ef06746R
ACTATTCTGAACACTGATGGACTTTTT
>84196c0e1674f1aR
AGGAAGCCAAGAAATACTGAAGaAACT
>4a1503929f4884cR
AAAAGGAAGCCAAGAAATACTGAAGaA
>08531cf06ce789aR
CaTTACAAAAGCAGGCTTGAGGAAATT
>5e7d0c157e2288dR
AGACaTTACAAAAGCAGGCTTGAGGAA
>7f5340f40f14c80R
CTGAGACaTTACAAAAGCAGGCTTGAG
>6e7161f4da2216dR
TCTCTGAGACaTTACAAAAGCAGGCTT
>e684c2b51e609f3R
GGCTCTCTGAGACaTTACAAAAGCAGG
>68c7db49dd902d8R
GAGGGCTCTCTGAGACaTTACAAAAGC
>a8c9de77519b649R
AATGAGGGCTCTCTGAGACaTTACAAA
>909f8f31c87fad6R
CCAAATGAGGGCTCTCTGAGACaTTAC
>bbf5932874e478bR
CTTCCAAATGAGGGCTCTCTGAGACaT
