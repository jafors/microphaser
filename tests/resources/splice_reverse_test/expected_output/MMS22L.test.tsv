id	transcript	gene_id	gene_name	chrom	offset	freq	nvar	nsomatic	nvariant_sites	strand	positions	aa_change
d30b9dac24c00d9R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231625	1	1	1	1	Reverse	97231644	p.Ser437Phe
c225e293273e48fR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231628	1	1	1	1	Reverse	97231644	p.Ser437Phe
7011a59454dda74R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231631	1	1	1	1	Reverse	97231644	p.Ser437Phe
1efa508d42bd9bcR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231634	1	1	1	1	Reverse	97231644	p.Ser437Phe
1fca23b4f4cddc8R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231637	1	1	1	1	Reverse	97231644	p.Ser437Phe
210cb330fa6bccfR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231640	1	1	1	1	Reverse	97231644	p.Ser437Phe
302a1fb3856f228R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231643	1	1	1	1	Reverse	97231644	p.Ser437Phe
9209752f31e1497R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231646	1	1	1	1	Reverse	97231644	p.Ser437Phe
4f610280fdd88a2R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231622	1	1	1	1	Reverse	97231644	p.Ser437Phe
96db41696eab759R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231619	1	1	1	1	Reverse	97231644	p.Ser437Phe
2b53a11462068baR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229240	1	1	0	1	Reverse	97229241	p.Thr564Met
bdaef3c47fab5c3R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229237	1	1	0	1	Reverse	97229241	p.Thr564Met
b342cc839483116R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229234	1	1	0	1	Reverse	97229241	p.Thr564Met
dde4785f31c14aeR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229231	1	1	0	1	Reverse	97229241	p.Thr564Met
9a11a0c3d529fd0R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229228	1	1	0	1	Reverse	97229241	p.Thr564Met
1ef3e87b4a95bdfR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229225	1	1	0	1	Reverse	97229241	p.Thr564Met
7db10c9ecbd341cR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229222	1	1	0	1	Reverse	97229241	p.Thr564Met
165b1f328bc037cR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229219	1	1	0	1	Reverse	97229241	p.Thr564Met
c805e1b269549fbR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97229216	1	1	0	1	Reverse	97229241	p.Thr564Met
2df200b2e3eb7d3R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231625	1	1	1	1	Reverse	97231644	p.Ser437Phe
6f698c636022ea8R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231628	1	1	1	1	Reverse	97231644	p.Ser437Phe
5de706d492a4e33R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231631	1	1	1	1	Reverse	97231644	p.Ser437Phe
8420c14e9383865R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231634	1	1	1	1	Reverse	97231644	p.Ser437Phe
997cb4e940b6d5fR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231637	1	1	1	1	Reverse	97231644	p.Ser437Phe
24362f4a2ca7fc6R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231640	1	1	1	1	Reverse	97231644	p.Ser437Phe
434939c4c42b55cR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231643	1	1	1	1	Reverse	97231644	p.Ser437Phe
e75bf7f0ef06746R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231646	1	1	1	1	Reverse	97231644	p.Ser437Phe
84196c0e1674f1aR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231622	1	1	1	1	Reverse	97231644	p.Ser437Phe
4a1503929f4884cR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231619	1	1	1	1	Reverse	97231644	p.Ser437Phe
08531cf06ce789aR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229240	1	1	0	1	Reverse	97229241	p.Thr564Met
5e7d0c157e2288dR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229237	1	1	0	1	Reverse	97229241	p.Thr564Met
7f5340f40f14c80R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229234	1	1	0	1	Reverse	97229241	p.Thr564Met
6e7161f4da2216dR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229231	1	1	0	1	Reverse	97229241	p.Thr564Met
e684c2b51e609f3R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229228	1	1	0	1	Reverse	97229241	p.Thr564Met
68c7db49dd902d8R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229225	1	1	0	1	Reverse	97229241	p.Thr564Met
a8c9de77519b649R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229222	1	1	0	1	Reverse	97229241	p.Thr564Met
909f8f31c87fad6R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229219	1	1	0	1	Reverse	97229241	p.Thr564Met
bbf5932874e478bR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97229216	1	1	0	1	Reverse	97229241	p.Thr564Met
